Products/ZenModel/migrate/addSolrService.py

```diff
@@ -0,0 +1,153 @@
+##############################################################################
+#
+# Copyright (C) Zenoss, Inc. 2017, all rights reserved.
+#
+# This content is made available according to terms specified in
+# License.zenoss under the directory where your Zenoss product is installed.
+#
+##############################################################################
+
+import json
+import logging
+import os
+log = logging.getLogger("zen.migrate")
+
+import Migrate
+import servicemigration as sm
+sm.require("1.0.0")
+from servicemigration import HealthCheck
+
+
+class AddSolrService(Migrate.Step):
+    """
+    Add Solr service and associated healthchecks.
+    """
+    version = Migrate.Version(116, 0, 0)
```

Comment:
Are we incrementing the migrate version every time we add a new migrate script? The highest migrate script in develop currently is 115